pkg/kubectl/cmd/get.go

```diff
@@ -69,6 +72,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			err := RunGet(f, out, cmd, args)
 			cmdutil.CheckErr(err)
 		},
+		ValidArgs: validArgs,
```

Comment:
This seems fine for bash completions, but it's not a complete list, as it doesn't include 'RESOURCE/NAME' args. Not a problem as they aren't used for validation, but we might want to note that in a comment, in case validation is added later.